erdjs/src/smartcontracts/argSerializer.spec.ts

```diff
@@ -54,27 +54,37 @@ describe("test serializer", () => {
         //     ],
         //     "0107@0000000200080009@abba@abba@abba"
         // );
+    });
+
+    it("should serialize <valuesToString> then back <stringToValues>: tuples", async () => {
+        serializeThenDeserialize(
+            ["tuple2<i32, i16>"],
+            [
+                Tuple.fromItems([new I32Value(100), new I16Value(10)])
+            ],
+            "00000064000a"
+        );
+    });
 
-        function serializeThenDeserialize(typeExpressions: string[], values: TypedValue[], joinedString: string) {
-            let types = typeExpressions.map(expression => typeParser.parse(expression)).map(type => typeMapper.mapType(type));
-            let endpointDefinitions = types.map(type => new EndpointParameterDefinition("foo", "bar", type));
+    function serializeThenDeserialize(typeExpressions: string[], values: TypedValue[], joinedString: string) {
```

Comment:
In `abiRegistry.ts`, you added parantheses around the parameters of lambda functions. Can you add them in the lambdas inside this function? It helps with code readability and it is a good idea.

Reply:
Done